packages/office-ui-fabric-react/src/components/pickers/BasePicker.tsx

```diff
@@ -194,10 +203,44 @@ export class BasePicker<T, P extends IBasePickerProps<T>> extends BaseComponent<
   }
 
   protected updateSuggestions(suggestions: any[]) {
-    this.suggestionStore.updateSuggestions(suggestions);
+    this.suggestionStore.updateSuggestions(suggestions, true);
     this.forceUpdate();
   }
 
+  protected onEmptyInputFocus() {
+    let suggestions: T[] | PromiseLike<T[]> = this.props.onInputFocus(this.state.items);
```

Comment:
Does there really need to be an onInputFocus callback like this? Why do this rather than something like call ` this.updateValue('', this.state.items) `